i18n/en/docusaurus-plugin-content-docs/current/get-started/overview.md

```diff
@@ -0,0 +1,172 @@
+
+- This methodology is for front-end projects only. If you're looking for a back-end architecture, consider [Clean Architecture][refs-clean-architecture].
+- A very simple app of a single page might not need the benefits of FSD and suffer from the overhead. However, FSD promotes a nice way of thinking, so feel free to use on the tiniest projects if you want.
+- A huge app, the size of the Google Cloud admin dashboard, will require a custom architecture. It could still be based on FSD, by the way.
+
+FSD doesn't enforce a particular programming language, UI framework or state manager — bring your own or see some [examples][refs-examples].
+
+If you have an existing project, fear not — FSD can be adopted incrementally. Just make sure that your team is **in&nbsp;pain** from the current architecture, otherwise a switch might not be worth it. <!-- For migration guidance, see the Migration section. TODO: add the migration section and link to it here -->
+
+## Basics
+
+In FSD, a [project consists][refs-splitting] of <mark>layers</mark>, each layer is made up of <mark>slices</mark> and each slice is made up of <mark>segments</mark>. 
+
+![themed--scheme](/img/visual_schema.jpg)
+
+The <mark>layers</mark> are standardized across all projects and vertically arranged. Modules on one layer can only interact with modules from the layers strictly below. There are currently seven of them (bottom to top):
+
+1. `shared` — reusable functionality, detached from the specifics of the project/business.
+   <small>(e.g. UIKit, libs, API)</small>
+2. `entities` — business entities.
+   <small>(e.g., User, Product, Order)</small>
+3. `features` — user interactions, actions that bring business value to the user.
+   <small>(e.g. SendComment, AddToCart, UsersSearch)</small>
+4. `widgets` — compositional layer to combine entities and features into meaningful blocks.
+   <small>(e.g. IssuesList, UserProfile)</small>
+5. `pages` — compositional layer to construct full pages from entities, features and widgets.
+6. `processes` — complex inter-page scenarios. 
+   <small>(e.g., authentication)</small>
+7. `app` — app-wide settings, styles and providers.
+
+
+Then there are <mark>slices</mark>, which partition the code by business domain. This makes your codebase easy to navigate by keeping logically related modules close together. Slices cannot use other slices on the same layer, and that helps with high cohesion and low coupling.
+
+Each slice, in turn, consists of <mark>segments</mark>. These are tiny modules that are meant to help with separating code within a slice by its technical purpose. The most common segments are `ui`, `model` (store,  actions), `api` and `lib` (utils/hooks), but you can omit some or add more, as you see fit.
+
+:::note
+
+In most cases, [it is recommended][ext-disc-api] to place `api` and `config` only in the shared layer
+
+:::
+
+**Also, FSD have few core-concepts:**
```

Comment:
> 2. The edits to the content of the intro make it harder to comprehend (to me personally)

> Also, FSD have few core-concepts

Partially understand, but you're welcome to suggest your version of this part about concepts :)

Reply:
> This filesystem feels like too much details, so I think we'd better wait with it. Also I think it'd be really nice as an interactive widget, not just a code block, but that's 💄 for the future.

Again, suggest your vision, how we could simplify files-structure for Overview :) 
(or where we should place this structure to easy-to-find user-cases)

Reply:
> See also

Agree, we could omit this part

Reply:
> Partially understand, but you're welcome to suggest your version of this part about concepts :)

But why do we need it?

Reply:
> But why do we need it?

Because it's not obvious for users at start

And **little note** about concepts would be useful for high-level understanding 🤷‍♂️ 

(only "layers-slices-segments" concept is not enough, even for "Overview")

Reply:
But why do users need to know this from the start? I can only speak for myself, but I don't think I'd want to see this info in the overview.

Public API — this is something that I would learn in the next page, "applying FSD in practice", when I would create an index file. The docs would say "this file is called a public API of the module, and it's an important part of isolation".

Isolation — kinda the same as public API. You don't need to know a lot of background knowledge to create an index file, so this is something that can be deferred, in my opinion

Domain Driven — yeah, it's important, but I think I'd rather it be mentioned when we introduce users to slices on that page, so that it would be in context

Reply:
fixed